docs/user-guide/layer-settings.md

```diff
@@ -92,25 +92,25 @@ In particular, the user is allowed to:
 
 ## Fields
 
-From this section of the *Settings* panel, [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows the user to add aliases to layer fields.
+From this section of the *Settings* panel, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows the user to add aliases to layer fields.
 
 <img src="../img/layer-settings/fields_settings.jpg" class="ms-docimage"  style="max-width:500px;"/>
 
-The panel lists the fields that make up the layer. For each field the following is highlighted:
+The panel shows the fields (feature attributes) of the layer: For each field the following are specified:
 
 * the **Name** of the field
 * the **Alias** of the field, which by default is empty
 * the **Type** of field
 
-The *Name* and the *Type* of the field cannot be modified, while the alias can be added for the user.
+The *Name* and the *Type* of the field cannot be modified, while the alias can be specified by the user.
 
-Using the **Localize** <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button, a popup is opened through which the user can add the translation of the field according to the language.
+Using the **Localize** <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button, a popup opens so that it is possible to configure the alias of the field as well as its translation.
```

Comment:
Using the **Localize** <img src="../img/button/localize_button.jpg" class="ms-docbutton"/> button, a popup opens so that it is possible to configure the alias of the field as well as its translations.